src/layouts/EditNavBar.jsx

```diff
@@ -738,7 +724,10 @@ const EditNavBar = ({ match }) => {
               )}
               <LoadingButton
                 isDisabled={hasErrors()}
-                onClick={() => saveNavData(siteName, originalNav, links, sha)}
+                onClick={async () => {
+                  await saveNavData({ originalNav, links, sha })
+                  window.location.reload()
+                }}
```

Comment:
side note: should we put the `window.location.reload` into the `saveNavData` hook's `onSuccess` property? this makes the invariant (refetch on success b/c of old API) maintained. if this is not done, we might run into saves w/o reloads so the user won't be able to read their own writes. we might also wish to add a `TODO` comment to remove the reload if it's shifted and/or file an issue pertaining to the removal of the `getNavData` and jot down the removal of the `reload` behaviour once the endpoint is migrated.